### PR TITLE
sentry release environment

### PR DESCRIPTION
**Pull request by obernardovieira**
NOTE: this was rebased from master. We should avoid doing it.

These are experimental changes. Not much to review. Just App.jsx

### CODE REVIEW

**Comment by obernardovieira** [comment]
@Jonathansoufer

**Comment by obernardovieira** [comment]
just added an extra change.

File: App.tsx [hunk 1]
```diff
@@ -139,12 +139,48 @@ Notifications.setNotificationHandler({
...
+        }),
+    ],
+    // sampleRate: 0.1,
+    // tracesSampleRate: 0.1,
+    tracesSampler: (samplingContext) => {
+        // Examine provided context data (including parent decision, if any) along
+        // with anything in the global namespace to compute the sample rate or
+        // sampling decision for this transaction
+
+        console.log('samplingContext', samplingContext.transactionContext.tags);
```
Comment (Jonathansoufer) [inline App.tsx:167]: Left over?

File: App.tsx [hunk 2]
```diff
@@ -139,12 +139,48 @@ Notifications.setNotificationHandler({
...
+                'donate' ||
+                samplingContext.transactionContext.tags['ipct-activity'] ===
+                    'claim')
+        ) {
+            return 1;
+        }
+        return 0.1;
+    },
+    beforeSend(event, hint) {
+        console.log('beforeSend', event, hint);
```
Comment (Jonathansoufer) [inline App.tsx:181]: What's the need for this console.log here?

**Review by Jonathansoufer — APPROVED** [review]
Besides console.logs, I don't see any reason to avoid merging it. We can remove it separately.

**Comment by obernardovieira** [comment]
console.logs were left on purpose here. Sorry, I should have mentioned. I don't know exactly what's going on on both of those calls. Maybe we should get a better logging system.

**Pull request merged**